core/src/main/scala/kafka/tools/ConsoleProducer.scala

```diff
@@ -96,23 +98,15 @@ object ConsoleProducer {
     props.put("serializer.class", config.valueEncoderClass)
     props.put("send.buffer.bytes", config.socketBuffer.toString)
     props.put("topic.metadata.refresh.interval.ms", config.metadataExpiryMs.toString)
-    props.put("client.id", "console-producer")
-
-    props
-  }
 
-  private def producerProps(config: ProducerConfig): Properties = {
```

Comment:
It fixes the problem we had with overridden client.id property. If you see the ConsoleConsumer code now they are quite the same (the consumer case is related to handle group.id). Am I missing something here ? :(